scripts/README.md

```diff
@@ -163,8 +163,14 @@ also overrides the value of the `TAG` variable as `v<version>`.
 it's empty and `master` HEAD will be considered the release branch.
     * `RELEASE_NOTES`: contains the filename with the release notes if `--release-notes`
 was passed. The release notes is a simple markdown file.
+    * `RELEASE_GCS`: contains the GCS bucket name to store the manifests if
```

Comment:
can we call this RELEASE_GCS_BUCKET instead? makes it more explicit that we are referring to the GCS bucket

Reply:
Makes sense. Done.